### PR TITLE
localstack host in process.env

### PR DESCRIPTION
**Pull request by usernameisunvaible**
Modified localstack host line in order to modify it with the environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,32 @@ A lightweight web interface to manage emails sent through [LocalStack SES](https
...
    - Open [http://localhost:3005](http://localhost:3005) in your web browser.
 
+### With Docker
+1. **Clone and Install:**
+   - Clone the repository: `git clone https://github.com/veertech/localstack-aws-ses-email-viewer.git`
+   - Navigate to the project directory: `cd localstack-aws-ses-email-viewer`
+2. **Build the Application:**
+   - run `docker build  . -t ses-viewer`
+3. **Run the Application:**
+   - run `docker run --rm --env LOCALSTACK_HOST=http://localstack:4566   -p 3005:3005 ses-viewer`
```
Comment (veelenga) [inline README.md:37]:
```suggestion
   - run `docker run --rm --env LOCALSTACK_HOST=http://localstack:4566 -p 3005:3005 ses-viewer`
```

**Review by usernameisunvaible — COMMENTED** [review]
ok

File: README.md [hunk 2]
```diff
@@ -13,18 +13,32 @@ A lightweight web interface to manage emails sent through [LocalStack SES](https
...
+   - Run: `export PORT=3005`
    - Run: `node server.js`
 
 3. **Access the Interface:**
    - Open [http://localhost:3005](http://localhost:3005) in your web browser.
 
+### With Docker
+1. **Clone and Install:**
+   - Clone the repository: `git clone https://github.com/veertech/localstack-aws-ses-email-viewer.git`
+   - Navigate to the project directory: `cd localstack-aws-ses-email-viewer`
```
Comment (veelenga) [inline README.md:33]: can we move this part above so it is not repeated for both sections? 

---

## Usage

- Clone the repository: `git clone https://github.com/veertech/localstack-aws-ses-email-viewer.git`
- Navigate to the project directory: `cd localstack-aws-ses-email-viewer`

### Run locally

- Install dependencies: `npm install`
- Run: `LOCALSTACK_HOST=http://localhost:4566 PORT=3005 node server.js`
- Open [http://localhost:3005](http://localhost:3005) in a web browser.

### Run with docker

- Build image: `docker build  . -t ses-viewer`
- Run: `docker run --rm --env LOCALSTACK_HOST=http://localstack:4566 -p 3005:3005 ses-viewer`
- Open [http://localhost:3005](http://localhost:3005) in a web browser.

**Review by veelenga — APPROVED** [review]

**Review by veelenga — APPROVED** [review]
Thank you

**Pull request merged**